packages/plugin-repo-cli/index.ts

```diff
@@ -245,7 +245,7 @@ async function commandBuild(args: CommandBuildArgs) {
 
 	chdir(previousDir);
 
-	const searchResults = (await execCommand('npm search joplin-plugin --searchlimit 5000 --json', { showStdout: false, showStderr: false })).trim();
+	const searchResults = (await execCommand('npm search keywords:joplin-plugin --searchlimit 5000 --json', { showStdout: false, showStderr: false })).trim();
```

Comment:
Follow-up PR: Consider using the API instead (or find some package that uses the API). Alternatively, look into sorting by last updated from the command line.